lib/app/console.rb

```diff
@@ -0,0 +1,242 @@
+
+  # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity
+  def account_menu
+    loop do
+      show_account_menu(@current_account)
+      case gets.strip
+      when COMMANDS[:show_cards]      then show_cards
+      when COMMANDS[:create_card]     then create_card
+      when COMMANDS[:destroy_card]    then destroy_card
+      when COMMANDS[:put_money]       then put_money
+      when COMMANDS[:withdraw_money]  then withdraw_money
+      when COMMANDS[:send_money]      then send_money
+      when COMMANDS[:destroy_account] then return destroy_account
+      when COMMANDS[:exit]            then exit
+      else warn_abount :wrong_command
+      end
+    end
+  end
+  # rubocop:enable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity
+
+  def create_account
+    loop do
+      account = Builder.build do |builder|
+        builder.input_name(input(:name))
+        builder.input_age(input(:age))
+        builder.input_login_credentials(input(:login), input(:password))
+        builder.input_empty_cards_list
+      end
+      next unless account
+
+      break Account.add @current_account = account
+    end
+  end
+
+  def load_account
+    return create_the_first_account if accounts.none?
+
+    attempts = 0
+    loop do
+      abort(I18n.t('ACCOUNT.ERRORS.LOGIN_ATTEMPTS')) if attempts >= AccountValidator::TRUSTED_LOGIN_ATTEMPTS_COUNT
+      credentials = obtain_account_credentials
+      attempts += 1
+      next warn_abount :unknown_account unless AccountValidator.account_exists?(credentials)
+
+      break setup_account(credentials[0])
+    end
+  end
+
+  def destroy_account
+    show('ACCOUNT.DESTROY_CONFIRMATION', with_invite: true)
+    Account.destroy(@current_account) if confirmed?
+  end
+
+  def create_the_first_account
+    show('ACCOUNT.FIRST_REQUEST', with_invite: true)
+    confirmed? ? create_account : run
+  end
+
+  def show_cards
+    return show('CARD.ERRORS.NO_CARDS_AVAILABLE') unless CardValidator.cards_available?(@current_account)
+
+    show_active_cards(@current_account)
+  end
+
+  def create_card
+    loop do
+      show_cards_creation_menu
+      card_type = gets.strip
+      return if card_type == COMMANDS[:exit]
+
+      next warn_abount(:wrong_card_type) unless CardValidator.card_type_valid?(card_type)
+
+      break Account.add_card(account: @current_account, type: card_type.intern)
+    end
+    Account.update(@current_account)
+  end
+
+  def destroy_card
+    choosen_card_index = convert_to_index choose_card
+    return if choosen_card_index.negative?
+
+    show(
+      'CARD.REMOVING_CONFIRMATION',
+      card_number: @current_account.cards[choosen_card_index].number,
+      with_invite: true
+    )
+    return unless confirmed?
+
+    Account.destroy_card(@current_account, choosen_card_index)
+    Account.update(@current_account)
+  end
+
+  def withdraw_money
+    choosen_card_index = convert_to_index choose_card
+    return if choosen_card_index.negative?
```

Comment:
Card index must be between 0 and cards_amount, isn't it?
Replace if I'm right, please